.github/workflows/actions.yml

```diff
@@ -273,7 +273,6 @@ jobs:
     - name: Run Tests
       run: |
         test/wasmBenchmarker/benchmark.py --engines $GITHUB_WORKSPACE/out/linux/x64/walrus --iterations 2 --verbose
-        test/wasmBenchmarker/benchmark.py --engines $GITHUB_WORKSPACE/out/linux/x64/walrus --iterations 2 --jit --verbose
```

Comment:
Is it the default now?

Reply:
Default all variation (interpreter, jit, jit-no-reg-alloc) are running, see at: https://github.com/Samsung/walrus/actions/runs/9597089499/job/26465412396